mautic/mautic
# TPROD-360 - CoreUpdate - Update - Migration PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


- Migrate PHP Templates to Twig
- Existing functionality should remain the same as the 5.x branch
- A few style updates were applied to keep everything consistent with the current mautic style

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. `/s/update` renders
3. `/s/update/schema` renders



- Testing steps similar to https://github.com/mautic/mautic/pull/10459
- I was able to get `/s/update` to render a new version is available by updating the `version` in https://github.com/mautic/mautic/blob/5.x/app/release_metadata.json to `4.0.0`, `rm -rf var/cache/*`, `bin/console mautic:update:find`
